tests/fixtures/compare_func_result1.json

```diff
@@ -6,11 +6,11 @@
     "root['Cars'][3]['production']"
   ],
   "values_changed": {
-    "root['Cars'][0]['dealers'][1]['quantity']": {
+    "root['Cars'][2]['dealers'][0]['quantity']": {
```

Comment:
The changes to the existing unittests are because the output is now relative to t2 instead of t1 as mentioned in the above comment.